resources/qml/PrintSetupSelector/Recommended/RecommendedSupportSelector.qml

```diff
@@ -87,82 +88,219 @@ Item
                 verticalCenter: parent.verticalCenter
             }
 
-            style: UM.Theme.styles.combobox_color
             enabled: recommendedPrintSetup.settingsEnabled
             visible: enableSupportCheckBox.visible && (supportEnabled.properties.value == "True") && (extrudersEnabledCount.properties.value > 1)
-            textRole: "text"  // this solves that the combobox isn't populated in the first time Cura is started
+            textRole: "name"  // this solves that the combobox isn't populated in the first time Cura is started
 
             model: extruderModel
 
-            property alias _hovered: supportExtruderMouseArea.containsMouse
-            property string color_override: ""  // for manually setting values
-            property string color:  // is evaluated automatically, but the first time is before extruderModel being filled
+            // knowing the extruder position, try to find the item index in the model
+            function getIndexByPosition(position)
             {
-                var current_extruder = extruderModel.get(currentIndex)
-                color_override = ""
-                if (current_extruder === undefined) return ""
-                return (current_extruder.color) ? current_extruder.color : ""
+                for (var item_index in model.items)
+                {
+                    var item = model.getItem(item_index)
+                    if (item.index == position)
+                    {
+                        return item_index
+                    }
+                }
+                return -1
+            }
+
+            onActivated:
+            {
+                if (model.getItem(index).enabled)
+                {
+                    forceActiveFocus();
+                    supportExtruderNr.setPropertyValue("value", model.getItem(index).index);
+                } else
+                {
+                    currentIndex = supportExtruderNr.properties.value;  // keep the old value
+                }
+            }
+
+            currentIndex: supportExtruderNr.properties.value
+
+            property string color: "#fff"
```

Comment:
Even if we override it later, the initial value here is only half of a (non-alpha) colour.